### PR TITLE
 When cannot find info, let user type them (Working version)

### PR DESCRIPTION
**Pull request by weathon**

### CODE REVIEW

File: src/python_autocite/__main__.py [hunk 1]
```diff
@@ -94,7 +94,15 @@ def main():
                 f.write("\n\n")
     else:
         for citation in formatted_citations:
-            print(citation)
+            tmp=str(citation)
+            # print(type(tmp))
```
Comment (thenaterhood) [inline src/python_autocite/__main__.py]:
```suggestion

```

File: src/python_autocite/__main__.py [hunk 2]
```diff
@@ -94,7 +94,15 @@ def main():
...
+            if("[[[AUTHORS]]]" in tmp): #可以优化速度
```
Comment (thenaterhood) [inline src/python_autocite/__main__.py]:
```suggestion
            if("[[[AUTHORS]]]" in tmp):
```

File: src/python_autocite/__main__.py [hunk 3]
```diff
@@ -94,7 +94,15 @@ def main():
...
+                tmp=tmp.replace("[[[AUTHORS]]]",input("Please enter author(s) manually: "))
+            if("[[[PUBLICATION DATE]]]" in tmp): #可以优化速度
```
Comment (thenaterhood) [inline src/python_autocite/__main__.py]:
```suggestion
            if("[[[PUBLICATION DATE]]]" in tmp):
```

File: src/python_autocite/__main__.py [hunk 4]
```diff
@@ -94,7 +94,15 @@ def main():
...
+                tmp=tmp.replace("[[[PUBLICATION DATE]]]",input("Please enter publication date manually: "))
+            if("[[[TITLE]]]" in tmp): #可以优化速度 yong tmp 也不行
```
Comment (thenaterhood) [inline src/python_autocite/__main__.py]:
```suggestion
            if("[[[TITLE]]]" in tmp):
```

File: src/python_autocite/__main__.py [hunk 5]
```diff
@@ -94,7 +94,15 @@ def main():
...
+                tmp=tmp.replace("[[[TITLE]]]",input("Please enter the title manually: "))#Forget the "=" python
```
Comment (thenaterhood) [inline src/python_autocite/__main__.py]:
```suggestion
                tmp=tmp.replace("[[[TITLE]]]",input("Please enter the title manually: "))
```

File: src/python_autocite/__main__.py [hunk 6]
```diff
@@ -94,7 +94,15 @@ def main():
...
+            print(tmp)#missing
```
Comment (thenaterhood) [inline src/python_autocite/__main__.py]:
```suggestion
            print(tmp)
```

**Review by thenaterhood — CHANGES_REQUESTED** [review]
Very cool! This adds something of value with a lot of potential that can be built on. Getting this wired in a little deeper would be nice, but this is a great place to start so I'm happy to accept it.

I left some suggestions because I don't think the code comments add value, but otherwise looks good.

**Comment by weathon** [comment]
I removed those comments

**Comment by thenaterhood** [comment]
Thanks! Looks good!

**Review by thenaterhood — APPROVED** [review]

**Pull request merged**